src/styles/platform-navigator.scss

```diff
@@ -0,0 +1,85 @@
+.platform-navigator {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  &__button {
+    display: flex;
+    justify-content: space-between;
+  }
+  &__dropdown {
+    border: 1px solid var(--amplify-colors-teal-80);
+    border-radius: 10px;
+    list-style-type: none;
+    padding: 0;
+    visibility: hidden;
+    z-index: 999;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    right: 0;
+    &--show {
+      visibility: visible;
+    }
+    &__item {
```

Comment:
Should we add some left/right padding for the items in the dropdown menu? I think it might give the items a little bit more room to breathe?

![image](https://github.com/aws-amplify/docs/assets/54393192/e2fa41ef-9750-4fb6-adc2-3bb84f9e0bae)

Reply:
Kinda like how the "home page" has some extra padding: 
![image](https://github.com/aws-amplify/docs/assets/54393192/83eb9b41-c5f9-43c7-b65e-d03c5224faa1)